packages/expo-face-detector/ios/EXFaceDetector.podspec

```diff
@@ -2,6 +2,11 @@ require 'json'
 
 package = JSON.parse(File.read(File.join(__dir__, '..', 'package.json')))
 
+firebase_sdk_version = '6.14.0'
+if defined? $FirebaseSDKVersion
+  firebase_sdk_version = $FirebaseSDKVersion
```

Comment:
Do you think adding a helpful message as we added in

https://github.com/expo/expo/blob/c17e2585d90ceeecd8b1a5c406e75528dfd9d271/packages/expo-image/expo-image.podspec#L5-L13

is worth it here too?

Reply:
Nah I wouldn't